component/authentication-endpoint/src/main/webapp/libs/jquery_3.6.0/jquery-3.6.0.js

```diff
@@ -1,15 +1,15 @@
 /*!
```

Comment:
Check if we can use the minified version.

Reply:
After the offline discussions with the team, it was decided to keep it as it is, given that this is a back-port fix and the same set of non-minified files are available in IS5.11.0 and public fix as well. Furthermore IS 5.7.0 will be having its end-of-lifecycle in August 2022. Hence the effort to change all JQuery js files to the corresponding minified versions across all 9 repositories involved will be not worth the time and effort.